src/Microsoft.AspNet.Authentication/DataHandler/Serializer/TicketSerializer.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.AspNet.Authentication.DataHandler.Serializer
 {
     public class TicketSerializer : IDataSerializer<AuthenticationTicket>
     {
-        private const int FormatVersion = 2;
+        private const int FormatVersion = 3;
```

Comment:
@Tratcher  @HaoK  @Eilon @tushargupta51 
You can design for this, I have done it multiple times.
When I designed .Net 4.5 to shim ClaimsPrincipal behind WindowsPrincipal, RolePrincipal and GenericPrincipal I had to design for .Net 4.0 and .Net 4.5 servers running in the same environment. A .Net 4.0 server would handle the new format and 4.0 handle the new.

The new .WriteTo OM in 4.6 is designed to flow through multiple versions without a version identifier.